medicines/document-manager/stub-api/README.md

```diff
@@ -0,0 +1,92 @@
+# Stub API
+
+## Purpose
+
+The stub API is built to unblock integration testing for Sentinel, **and has
+no practical functionality**. It does not interact with the Sentinel server
+or Azure Blob storage in any way, it simply uses in-memory storage to mock
+the actual API functionality.
+
+The intended usage pattern is as follows:
+
+- Send a DELETE or POST request for a document;
+- Continue to poll the GET endpoint for that document until the status has
+updated.
+
+In the background, the real API will asynchronously process the request and
+update the status of the document as it progresses.
+
+## Running
+
+**NOTE: THIS STILL NEEDS TO BE CONTAINERIZED.**
```

Comment:
looks like the Dockerfile runs `stub.pl`, so can the Docker run instructions be added now?

Reply:
Yes.